renderer/src/common/lib/constants.ts

```diff
@@ -3,3 +3,14 @@ export const META_MCP_SERVER_NAME = 'toolhive-mcp-optimizer'
 export const ALLOWED_GROUPS_ENV_VAR = 'ALLOWED_GROUPS'
 export const MCP_OPTIMIZER_REGISTRY_SERVER_NAME = 'mcp-optimizer'
 export const DEPRECATED_MCP_OPTIMIZER_REGISTRY_SERVER_NAME = 'meta-mcp'
+
+/**
+ * Determines if the platform is probably using native containers.
+ * Native containers mean Docker runs directly on the host OS (Linux),
+ * as opposed to running in a VM (macOS/Windows with Docker Desktop).
+ * When native containers are used, host networking mode allows direct
+ * access to the host's network stack.
+ */
+export function isProbablyUsingNativeContainers(): boolean {
+  return window.electronAPI.isLinux
+}
```

Comment:
I would remove this from here, this is only for constant, and considering that this fn is used only on a single file (`renderer/src/features/meta-mcp/hooks/use-create-optimizer-workload.ts`) you can call directly`window.electronAPI.isLinux`

Reply:
but this is what you're doing here, you are calling isLinux, and also you put a function inside a constant file, at least you cna move this fn  where it is used, cause it is used only in a specific file, so it shouldn't be in common, cause it is not shared with multiple files

Reply:
true, I can move it into the same file. and I can improve the documentation to actually explain what is happening there

Reply:
pushed